Content.Server/Anomaly/Effects/EntityAnomalySystem.cs

```diff
@@ -68,15 +72,22 @@ private void SpawnMonstersOnOpenTiles(EntitySpawnAnomalyComponent component, Tra
             {
                 if (!physQuery.TryGetComponent(ent, out var body))
                     continue;
-                if (body.BodyType != BodyType.Static ||
-                    !body.Hard ||
-                    (body.CollisionLayer & (int) CollisionGroup.Impassable) == 0)
+
+                if (component.WallEntity == true && body.BodyType != BodyType.Static)
                     continue;
+
+                if (component.WallEntity == false)
+                {
+                    if (body.BodyType != BodyType.Static ||
+                        !body.Hard ||
+                        (body.CollisionLayer & (int) CollisionGroup.Impassable) == 0)
+                        continue;
+                }
```

Comment:
what's the practical difference that requires this?

Reply:
stops walls spawning on tables and in doorways, not a big issue gameplay wise but looks like ass

Reply:
I decided to remove this bool in the end, it causes other issues and to get it perfect would require a very lengthy if statement.